app/workers/Cataloger.scala

```diff
@@ -296,14 +296,14 @@ object Cataloger {
     val coll = Collection.findById(item.collectionId).get
     val resmap = ResourceMap.findById(coll.resmapId).get
     val cataloger = new Cataloger(resmap, Store.content(item))
-    val ctype = ContentType.findById(item.ctypeId).get
+    val rmap = ResourceMap.findById(coll.resmapId).get
```

Comment:
If we go in this direction, I'll remove this line and use the `resmap` defined above (didn't notice it before).